examples/codegen-react-apollo/nhost/config.yaml

```diff
@@ -0,0 +1,138 @@
+metadata_directory: metadata
+services:
+  hasura:
+    version: v2.9.0-beta.1
```

Comment:
we're trying to pin hasura, hasura-storage and hasura-auth versions to the ones used in our cloud. Although there's no e2e test on this example, I would use the same versions as everywhere else:
```yaml
services:
  mailhog:
    port: 8025
  hasura:
    version: v2.8.0
    environment:
      hasura_graphql_enable_remote_schema_permissions: false
  auth:
    version: 0.9.1
  storage:
    version: 0.2.2
```